cmd/juju/caas/add.go

```diff
@@ -44,7 +46,7 @@ Adds a k8s endpoint and credential to Juju.`[1:]
 
 var usageAddCAASDetails = `
 Creates a user-defined cloud and populate the selected controller with the k8s
-cloud details.
+cloud details. Speficify non default kubeconfig file location using $KUBECONFIG environment variable.
```

Comment:
Mention you can pipe from stdin also.
Should also explain what a context is, ie the config file can contain definitions for different k8s clusters and the context arg is used to pick which one to use. Perhaps the arg should be "cluster-name" to be more user friendly.